<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualizador</class>
 <widget class="QWidget" name="Visualizador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelCcds">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>CCDs:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCuentas">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Cuentas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFondoCielo">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Fondo De Cielo:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQnx1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>QNX1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQnx2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>QNX2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQnx3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>QNX3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQnx4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>QNX4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCcd4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CCD4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCcd1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CCD1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCcd2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CCD2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCcd3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CCD3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValor">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor = </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValorX">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X = </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValorY">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y = </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMu">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSigma">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValorEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValorXEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValorYEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>1011</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
